src/controller.js

```diff
@@ -0,0 +1,49 @@
+const processWeatherdesc = (condition) => {
```

Comment:
**Optional tips:** Switch is the best choice for handling this operation. If you want then you can implement switch.